pkg/manager/cron/cron.go

```diff
@@ -41,8 +41,23 @@ func (c Cron) processEdits() {
 				voteThreshold = 1
 			}
 
-			var err error
-			if e.VoteCount >= voteThreshold {
+			editVotes, err := c.rfp.Repo().Edit().GetVotes(e.ID)
```

Comment:
The cron now has to go and get the votes for each edit, but it's only happening every few minutes, and there shouldn't be more than reasonable amount of edits to handle each run, so I think this is ok.